### PR TITLE
Updated readme

### LINKED ISSUE #66: To Add No of issues/pull requests/forks/watchers/stars in Readme file
It should be added at the top, just after the first title
Reference: https://shields.io/category/issue-tracking

### PR DESCRIPTION
**Pull request by Samridhi-98**
# Description

Updated `README.md` with badges and also made little UI changes. 

Fixes #66 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Screenshot

op-ss

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@Manvityagi  Please review my PR.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
-## Opportunity Calendar
+## Opportunity Calendar <img src="https://github.com/Samridhi-98/Images/blob/master/Images/op-logo.png" width="3%">
 
+<p>
+
+[![GitHub contributors](https://img.shields.io/github/contributors/adityabisoi/REST-fetch)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/graphs/contributors/)
```
Comment (Manvityagi) [inline README.md]: We don't need this one as of now

File: README.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+[![Issues](https://img.shields.io/github/issues/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/issues)
+[![PRs](https://img.shields.io/github/issues-pr/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/pulls)
+[![Forks](https://img.shields.io/github/forks/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Stars](https://img.shields.io/github/stars/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Visits Badge](https://badges.pufler.dev/visits/Girl-Code-It/Opportunity-Calendar-Backend)](https://badges.pufler.dev)
```
Comment (Manvityagi) [inline README.md:9]: We don't need this one as of now

File: README.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
 [![Build Status](https://www.travis-ci.com/Girl-Code-It/Opportunity-Calendar-Backend.svg?branch=develop)](https://www.travis-ci.com/Girl-Code-It/Opportunity-Calendar-Backend)
 
+</p>
+
 Opportunity Calendar is the one-stop place to refer important opportunities available in tech-space like newly posted jobs, internships, hackathons, tech-conferences, scholarships etc.
 
 The frontend of the same project can be found [here](https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend)
 
-## Table of Contents
+## Table of Contents <img src="https://github.com/Samridhi-98/Images/blob/master/Images/rocket.svg" width="3%">
```
Comment (Manvityagi) [inline README.md]: These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.

File: README.md [hunk 4]
```diff
@@ -16,7 +26,7 @@ The frontend of the same project can be found [here](https://github.com/Girl-Cod
 6. [Contributing](#contributing)
 7. [Copyright and license](#copyright-and-license)
 
-<h2 align="center">Install</h2>
+## Install <img src="https://github.com/Samridhi-98/Images/blob/master/Images/test.svg" width="3%">
```
Comment (Manvityagi) [inline README.md]: These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.

File: README.md [hunk 5]
```diff
@@ -41,8 +51,9 @@ $ npm run start:dev
 # Visit http://localhost:3030/ in your browser
 ```
 
-<h2 align="center">Introduction</h2>
-Opportunity Calendar provides instant information on newly posted opportunuties for techies. There are times when applicants miss deadlines or get to know about available opportunties only after the deadline of their application passes. 
+## Introduction <img src="https://github.com/Samridhi-98/Images/blob/master/Images/op-about.png" width="3%">
```
Comment (Manvityagi) [inline README.md]: These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.

**Review by Manvityagi — CHANGES_REQUESTED** [review]
The images that you have added in the readme, please upload them in the `assets` folder at the root of this project and then use them.

File: README.md (hunk 2 above)
Comment (Samridhi-98) [reply README.md:9]: Is it wrong? 
Do we have to add shield badges naah?

File: README.md (hunk 3 above)
Comment (Samridhi-98) [reply README.md]: okay, I will add them to the assets folder.

**Comment by Samridhi-98** [comment]
@Manvityagi  Updated as you suggested please review.

**Review by Manvityagi — APPROVED** [review]

**Pull request merged**